src/main/resources/assets/securitycraft/lang/es_es.json

```diff
@@ -0,0 +1,768 @@
+	"gui.securitycraft:scManual.intro.2": "de Geforce y bl4ckscor3",
+	"gui.securitycraft:scManual.author": "Traducido por Ryo567",
+	"gui.securitycraft:scManual.ownableBlock": "Bloque con propietario: Este bloque es resistente a explosiones y solo puede ser eliminado por el jugador que lo colocó usando el eliminador de bloques universal.",
+	"gui.securitycraft:scManual.passwordProtectedBlock": "Protección con contraseña: Este bloque solo puede utilizarse si un jugador introduce la contraseña correcta.",
+	"gui.securitycraft:scManual.viewActivatedBlock": "Escáner de retina: Este bloque se activa pasando el cursor por encima a una distancia inferior a 6 bloques.",
+	"gui.securitycraft:scManual.explosiveBlock": "Explosivo: Este bloque puede detonarse a distancia, así como ser activado y desactivado con el control remoto de minas.",
+	"gui.securitycraft:scManual.customizableBlock": "Personalizable: Puedes usar el modificador de bloques universal y los módulos de SecurityCraft para personalizar las característcas de este bloque.",
+    "gui.securitycraft:scManual.options": "Opciones disponibles:",
+    "gui.securitycraft:scManual.modules": "Módulos disponibles:",
+	"gui.securitycraft:scManual.recipe.keypad": "Coloca un marco y haz clic con un panel numérico.",
+	"gui.securitycraft:scManual.recipe.keypad_chest": "Coloca un cofre y haz clic derecho con el panel numérico.",
+	"gui.securitycraft:scManual.recipe.keypad_furnace": "Coloca un horno y haz clic derecho con el panel numérico.",
+	"gui.securitycraft:scManual.recipe.reinforced": "Los bloques reforzados se pueden fabricar de la siguiente manera, si no se especifica lo contrario en el libro. Coloca el equivalente estándar (de la piedra reforzada, sería la piedra) y rómpelo con el reforzador de bloques universal o haz clic derecho con el reforzador de bloques universal, colócalo en el hueco y cierra la interfaz.",
+	"gui.securitycraft:scManual.recipe.reinforced_iron_trapdoor": "Los bloques reforzados se pueden fabricar de la siguiente manera, si no se especifica lo contrario en el libro. Coloca el equivalente estándar (de la piedra reforzada, sería la piedra) y rómpelo con el reforzador de bloques universal o haz clic derecho con el reforzador de bloques universal, colócalo en el hueco y cierra la interfaz.",
+	"gui.securitycraft:scManual.recipe.adminTool": "La herramienta de administrador solo está disponible en modo creativo. Debe activarse en el archivo de configuración.",
+	"gui.securitycraft:scManual.recipe.admin_tool": "La herramienta de administrador solo está disponible en modo creativo. Debe activarse en el archivo de configuración.",
+	"gui.securitycraft:scManual.recipe.reinforced_crystal_quartz": "Fabricado con bloques de cuarzo cristal normales con un reforzador de bloques universal.",
+	"gui.securitycraft:scManual.recipe.sc_manual": "El manual de SecurityCraft solo está disponible en el inventario creativo o escribiendo «/sc help».",
```

Comment:
Yes, that is true. The language file has a couple lines that are outdated and no longer used.

Reply:
I skipped the «(sin las comillas)» explanation because I assumed it's without the quotes, but you're right, better to not skip it. Sorry! :)

Reply:
Unresolving conversation until we figure out the quotes issue

Reply:
Well, I'm changing because although it is translated with « » in 1.15 and 1.16, when I force the string to appear ingame, it appears as English quote (").